### ISSUE TITLE
yuedajiong-question #04:  can we use this iJEPA pre-train for that LeCun total-JEPA?

### ISSUE THREAD

**Issue by yuedajiong** [facebookresearch/ijepa#9]
just from my understanding:

1. the loss fulcrum is mask in iJEPA.
2. the LeCun total-JEPA is still E2E differentiable, but maybe trained in a mini interactive env based vision, and feedback signal is from action but not direct vision.

can we directly use this pre-train parameters?  or re-train from zero?

in other word:  the representation of mask-fulcrum-self-supervised , and,  the representation of sequencial-actions-fulcrum-self-supervised in later total-JEPA,  are similar?

**Comment by mikerabbat** [comment]
Hi, thanks for your interest. This is a first work towards building out Yann's full vision of the JEPA, but there is plenty of additional work to be done. In this work we learn representations for images. The model (encoder and predictor) are trained in an end-to-end manner.
